activerecord/lib/active_record/connection_adapters/sqlite3/schema_statements.rb

```diff
@@ -2,6 +2,41 @@ module ActiveRecord
   module ConnectionAdapters
     module SQLite3
       module SchemaStatements # :nodoc:
+        # Returns an array of indexes for the given table.
```

Comment:
Yep, I guess a separate commit in the PR is fine. My preference would be to leave things the way they are, since, just as an example, moving `#indexes` for the SQLite3 adapter is kind of a cosmetic change, which we generally try and avoid 😢